modules/vba_documents/lib/tasks/one_time_status_update_20200803.rake

```diff
@@ -0,0 +1,2210 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/BlockLength
+namespace :vba_documents do
```

Comment:
I would love to see this in the `:tmp` namespace personally -- easier to be sure to remove it later since intention is clear. But 🤷 not going to push too hard.

Reply:
I actually agree, maybe a :vba_documents followed by a :tmp so that we know what this was intended to do but still mark it as a temp rake task.

Reply:
@johnpaulashenfelter , what's easier to cleanup later:
```
namespace :vba_documents do
  namespace :temp do
```
or
```
namespace :temp do
  namespace :vba_documents do
```
or is either one good?

Reply:
Personally i like temp as the namespace but both communicate intent. Since this is down in a module my normal assumption of having a `tmp.rake` is a bit less strong.